lib/src/git_backend.rs

```diff
@@ -578,6 +579,19 @@ fn commit_from_git_without_root_parent(
             .map(|oid| CommitId::from_bytes(oid.as_bytes()))
             .collect_vec()
     };
+    // If the commit is a conflict, the conflict labels are stored in a commit
+    // header separately from the trees.
+    let conflict_labels: ConflictLabels = commit
+        .extra_headers()
+        .find(JJ_CONFLICT_LABELS_COMMIT_HEADER)
+        .map(|header| {
```

Comment:
nit: extract a method for the conversion similar to `root_tree_from_git_extra_header()`

Actually, perhaps it would be more readable to also refactor the existing `root_tree_from_git_extra_header()` to take `commit` as input?